config/cops.yml

```diff
@@ -1 +1,24 @@
-Metrics/ClassLength: 400_000
+Metrics/AbcSize:
+  violation_cost: 100_000
```

Comment:
I was somewhat using them interchangeably. Do we prefer to rename these to points?